CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ x.x.x
 -----
 
 * Updates UNTL_USAGE_LINK setting to point to current help pages.
+* Fixes a couple flake8 issues relating to string comparisons.
```

Comment:
technically 3 is more than a couple

Reply:
Now there's a familiar refrain! I get that a _lot_